src/components/inputs/localized-rich-text-input/editor.js

```diff
@@ -177,8 +178,10 @@ const Editor = props => {
 
 // eslint-disable-next-line react/display-name
 const renderEditor = (props, editor, next) => {
+  const internalId = `${props.id}__internal__id`;
+
   const children = React.cloneElement(next(), {
-    tagName: 'output',
+    id: internalId,
```

Comment:
we don't want two elements with the same id, so we change the `<div contenteditable>` id to an internal id